index.html

```diff
@@ -416,6 +417,15 @@ <h2><dfn>getEntriesByName()</dfn> method</h2>
           </section>
         </section>
       </section>
+      <section>
+        <h2><dfn>takeRecords()</dfn> method</h2>
+        <p>The <a>takeRecords()</a> method instructs the user agent to flush  entries in <a>context object</a>'s <a>observer buffer</a> and must run these steps:</p>
+        <ol data-link-for="PerformanceObserver">
+          <li>Let <var>entries</var> be a copy of <a>context object</a>'s <a>observer buffer</a>.</li>
+          <li>Empty <a>context object</a>'s <a>observer buffer</a>.</li>
+          <li>Return <var>entries</var>.</li>
```

Comment:
Don't we need to convert the list of PerformanceEntry objects to a PerformanceEntryList?

Reply:
Those are equivalent "typedef sequence<PerformanceEntry> PerformanceEntryList;"

That said, agree that this should be clearer, will update the definition.

Reply:
Oops, that makes sense, thanks!